service/database/index.js

```diff
@@ -87,7 +87,10 @@ class Mongo {
             const res = await collection.insertMany(data);
             return res;
         } catch (e) {
-            if (silent) {
+            if (e.code === 11000) { 
```

Comment:
We should probably only silently return if 'silent' is true

Reply:
Yes true, I have handled that case in else if statement.